src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -390,10 +392,7 @@ def fn(
                         comparison_delta,
                     )
                     has_other_data = self.check_if_results_have_data(original_results)
-                    if isinstance(original_results, SnubaTSResult):
-                        dataset_meta = original_results.data.get("meta", {})
-                    else:
-                        dataset_meta = list(original_results.values())[0].data.get("meta", {})
```

Comment:
Why was the `else` block removed? This is to handle the multi-axis charts right?

Reply:
lol probably should've drafted this sorry, I forgot what this block was and wanted CI to explain it

Reply:
I'm not sure if its multi-axis though, when i ran the events_stats.py tests locally it passed 🤔

Reply:
Hm... could we just be missing test coverage? This code reads like it's taking the meta from the first timeseries.

Reply:
😨 if that's the case that's much worse lol

Reply:
🤔 finally tested this, yup its multi-axis but its not tested in events-stats 😬

Reply:
Can we get a test added here? 😅

Reply:
I'll add them in a separate PR, don't want to overload this one